src/Pages/ShowAparelhoRelatorio.jsx

```
@@ -0,0 +1,76 @@
+import TopBar from "../Components/TopBar";
+import MenuBar from "../Components/MenuBar";
+import ShowInfo from "../Components/ShowInfo";
+import FeedbackPopup from "../Components/FeedbackPopup";
+
+function NumDisponiveis(aparelhos) {
+    count = 0
+    foreach (aparelho in aparelhos) 
+    {
+        if (aparelho.disponibilidade == "Disponível")
+            count++
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)